packages/components/src/components/Modal/components/Backdrop.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+    position: fixed;
+    z-index: 10000;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background: rgba(0, 0, 0, 0.3);
+    backdrop-filter: blur(5px);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    overflow: auto;
+    > :first-child {
+        margin-top: auto;
+    }
+    > :last-child {
+        margin-bottom: auto;
+    }
+`;
+
+export type BackdropProps = {
+    onClick?: () => void;
+    children: React.ReactNode;
+    className?: string;
```

Comment:
`className` is used under the hood for `styled(Backdrop)`.

Otherwise good catch, I replaced the old Backdrop component with the new one, and it behaves even better imo.

https://github.com/trezor/trezor-suite/pull/5132/commits/044ea9d068d1f1fdde36a42d728d577d13449939

Reply:
👍 

not sure if related to this change
<img width="1792" alt="Screenshot 2022-03-24 at 13 26 12" src="https://user-images.githubusercontent.com/33235762/159916531-aeef1082-eec8-43bf-8d77-738d437f5eba.png">